house_of_refuge/main/apps.py

```diff
@@ -5,4 +5,4 @@
 class MainConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "house_of_refuge.main"
-    verbose_name = _("Grupa Zasoby")
+    verbose_name = "House of Refuge"
```

Comment:
Why not make this translatable as well?

Reply:
I decided to treat this as a proper noun ("nazwa własna") - the name of the application.

Reply:
I see your point, but it's also quite handy for non-technical users to see a more familiar name in Django admin...

Reply:
But what would be the translation that Polish translation? Hardcoding "Grupa Zasoby" as a translation of "House of Refuge" would not only be clunky but also wouldn't be good because you said that other Polish groups are looking to possibly use the software.

Maybe you want to make this configurable in settings?

Reply:
yeah, that would probably be the best option! we can do it in the separate PR perhaps